File: Views/src/app/_components/administrative/local/providers/supplier/dto/supplier-dto.ts
import { TypePaymentDto } from "src/app/_components/administrative/local/financial/components/type-payment/dto/type-payment-dto";
import { AddressDto } from "src/app/_shared/dtos/address-dto";
import { ContactDto } from "src/app/_shared/dtos/contact-dto";


export class SupplierDto {
  id: number;
  name: string;
  seller: string;
  description: string;
  operation: string;
  addressId: number;
  address: AddressDto;
  contactId: number;
  contact: ContactDto;
  typespayments: TypePaymentDto[];
  toSeach:string;
}

File: Services/Dto/SupplierDto.cs
using System.Collections.Generic;
using Services.Dto.Financial;

namespace Services.Dto
{
    public class SupplierDto
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Seller { get; set; }
        public string Description { get; set; }
        public string Operation { get; set; }
        public int AddressId { get; set; }
        public AddressDto Address { get; set; }
        public int ContactId { get; set; }
        public ContactDto Contact { get; set; }
        public List<TypePaymentDto> typespayments { get; set; }
        public string ToSeach { get; set; }
    }
}